okapi-core/src/main/raml/okapi.raml

```diff
@@ -790,10 +815,18 @@ types:
           must be deployed (optionally) and enabled/disabled by the existing
           tenants-modules CRUD service.
         queryParameters:
+          async:
+            description: Whether to install in the background
+            type: boolean
+            required: false
           deploy:
             description: Whether to deploy (or undeploy if disabling)
             type: boolean
             required: false
+          ignoreErrors:
+            description: Whether to ignore module init errors
```

Comment:
Should be more verbose what "ignore" means. The complete explanation that guide.md should be included in the description here to show on the primary documentation site https://dev.folio.org/reference/api/#okapi

Reply:
The RAML will never include all material from the guide. But I can copy the material for this one if you like.

Reply:
Yes, please.